### ISSUE TITLE
Attribution box should be keyboard-accessible 

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/universalviewer#3]
A user navigating the viewer with a keyboard cannot gain focus of the Attribution box, and thus cannot close it without using a mouse.

**Comment by snydman** [comment]
Should there also be mouse and keyboard controls to toggle this on and off later in the same session for the same object?

**Comment by edsilv** [comment]
If you add `tabindex="0"` to the attribution close button it should be accessible in the tab order.

**Comment by edsilv** [comment]
@snydman It's possible that we could minimise instead of hide the attribution div entirely.

**Comment by mejackreed** [comment]
Resolved upstream
